src/Equations/damaged-elastic/Kernels/Time.cpp

```diff
@@ -183,18 +183,21 @@ void seissol::kernels::Time::computeAder(double i_timeStepWidth,
   krnl.spaceTimePredictor = stp;
   krnl.spaceTimePredictorRhs = stpRhs;
   krnl.execute();
-#else  // USE_STP
-
-  real epsInitxx = damagedElasticParameters->epsInitxx;
-  real epsInityy = damagedElasticParameters->epsInityy;
-  real epsInitzz = damagedElasticParameters->epsInitzz;
-  real epsInitxy = damagedElasticParameters->epsInitxy;
-  real epsInityz = damagedElasticParameters->epsInitxx;
-  real epsInitzx = damagedElasticParameters->epsInitxx;
-
-  real const damageParameter = data.material.local.Cd;
-  real const breakCoeff = damagedElasticParameters->scalingvalue * damageParameter;
-  real const betaAlpha = damagedElasticParameters->beta_alpha;
+#else // USE_STP
+
+#ifdef USE_DAMAGEDELASTIC
+  const real epsInitxx = damagedElasticParameters->epsInitxx; // eps_xx0
+  const real epsInityy = damagedElasticParameters->epsInityy; // eps_yy0
+  const real epsInitzz = damagedElasticParameters->epsInitzz; // eps_zz0
+  const real epsInitxy = damagedElasticParameters->epsInitxy; // eps_xy0
+  const real epsInityz = damagedElasticParameters->epsInityz; // eps_yz0
+  const real epsInitzx = damagedElasticParameters->epsInitzx; // eps_zx0
+
+  real const damagedParameter = data.material.local.Cd; // 1.2e-4*2;
```

Comment:
Is it a parameter describing the damage (damageParameter) or is the parameter damaged?

Reply:
I don't precisely know here. May be a better name would be damageParameter?

Reply:
I think `damageParameter` is the better choice. It is written this way in other places.

Reply:
Done. Thank you